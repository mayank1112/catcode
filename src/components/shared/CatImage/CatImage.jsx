import { catImageUrl } from "../../../utils/paths";

const CatImage = ({ catId }) => (
    <img alt='Cat' src={catImageUrl(catId)} />
);

export default CatImage;
