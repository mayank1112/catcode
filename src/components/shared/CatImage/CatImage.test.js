import Enzyme, { shallow } from 'enzyme';
import CatImage from './CatImage';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

Enzyme.configure({ adapter: new Adapter() });

it('<CatImage> component should match snapshot', () => {
    const component = shallow(<CatImage />);
    expect(toJson(component)).toMatchSnapshot();
});
