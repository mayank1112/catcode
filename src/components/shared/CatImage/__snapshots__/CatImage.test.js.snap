// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CatImage> component should match snapshot 1`] = `
<img
  alt="Cat"
  src="http://quantcats.herokuapp.com/static/cats/undefined.png"
/>
`;
